torch/distributed/_sharded_tensor/__init__.py

```diff
@@ -124,7 +126,8 @@ def rand(sharding_spec: ShardingSpec,
          process_group=None,
          init_rrefs=False):
     """
-    Creates a rand :class:`ShardedTensor`. Needs to be called on all ranks in an SPMD fashion.
+    Returns a :class:`ShardedTensor` filled with random numbers from a uniform distribution on the
+        interval [0, 1)[0,1). Needs to be called on all ranks in an SPMD fashion.
```

Comment:
nit: You actually want to use the following docstring here: ":math:`[0, 1)`". See https://github.com/pytorch/pytorch/blob/master/torch/_torch_docs.py#L7756.

Reply:
great ptr.